### ISSUE TITLE
allinone NullPointerException

### ISSUE THREAD

**Issue by virtuald** [batfish/batfish#206]
When running allinone.Main from eclipse, if I try to do a simple query it fails with a NullPointerException:

```
batfish> get #checkreachability 10.0.4.200 tcp
get #checkreachability 10.0.4.200 tcp
java.lang.NullPointerException
	at org.batfish.client.QuestionHelper.getReachabilityQuestion(QuestionHelper.java:103)
	at org.batfish.client.QuestionHelper.resolveMacro(QuestionHelper.java:168)
	at org.batfish.client.Client.answerType(Client.java:730)
	at org.batfish.client.Client.get(Client.java:1254)
	at org.batfish.client.Client.processCommand(Client.java:2040)
	at org.batfish.client.Client.processCommand(Client.java:1985)
	at org.batfish.client.Client.runInteractive(Client.java:2319)
	at org.batfish.client.Client.run(Client.java:2272)
	at org.batfish.allinone.AllInOne.run(AllInOne.java:117)
	at org.batfish.allinone.Main.main(Main.java:6)
```

I suspect I just don't have my classpath or some environment setup correctly, but it seems like it would be nice if there were a better error message if that were the case. :)

**Comment by virtuald** [comment]
When not running from the shell environment, you need to provide the `-DbatfishQuestionPluginDir` system property to the JVM. In eclipse, I add this to the run configuration 'VM Arguments'.

```
-DbatfishQuestionPluginDir=${workspace_loc:/question}/target
```

It seems that if the question plugins are required (surely they are?), that batfish should exit with an error (and mention to set this property) if one is not configured.

**Comment by ratulm** [comment]
Yes, most of the questions require that plugin. We'll make the error message more helpful. 

While you are update the WiKi for eclipse, feel free to add this point too. 

Thanks.

**Comment by dhalperi** [comment]
Want to send a PR?

**Comment by ratulm** [comment]
Fixed by 4d4e1396f64286a8579428f71a7f6e307c9485bb
